UniswapV3Adapter/contracts/lib/Maintainable.sol

```diff
@@ -0,0 +1,57 @@
+// SPDX-License-Identifier: MIT
+
```

Comment:
Please order functions layout by type
Refer https://docs.soliditylang.org/en/v0.8.17/style-guide.html#order-of-layout